uamqp/async_ops/connection_async.py

```diff
@@ -98,6 +98,9 @@ def __init__(self, hostname, sasl,
     async def _close_async(self):
         _logger.info("Shutting down connection %r.", self.container_id)
         self._closing = True
+        if self._cbs:
+            await self.auth.close_authenticator_async()
```

Comment:
loop?

Reply:
loop is not used in `close_authenticator_async`